packages/govuk-frontend/src/govuk/init.jsdom.test.mjs

```diff
@@ -143,6 +175,42 @@ describe('initAll', () => {
       })
     )
   })
+
+  it('executes onError callback on component create if specified', () => {
+    document.body.classList.add('govuk-frontend-supported')
+    document.body.innerHTML = '<div data-module="govuk-accordion"></div>'
+
+    jest.mocked(GOVUKFrontend.Accordion).mockImplementation(() => {
+      throw new Error('Error thrown from accordion')
+    })
+
+    const errorCallback = jest.fn((error, context) => {
+      console.log(error)
+      console.log(context)
+    })
+
+    // Silence warnings in test output, and allow us to 'expect' them
+    jest.spyOn(global.console, 'log').mockImplementation()
```

Comment:
**nitpick** Missed that during the review of `createAll`, so one for a separate PR as well, but seems `jest.spyOn` does not clean up after itself and [we may need to call `jest.restoreAllMocks`](https://jestjs.io/docs/jest-object#jestspyonobject-methodname) to ensure the mock is cleaned up for further tests.

Reply:
Added [a small story to track this](https://github.com/alphagov/govuk-frontend/issues/5304) as we should have a look more widely than the `console.log` spies for the `createAll` and `initAll` tests.